test/package/packageConvert.test.ts

```diff
@@ -31,9 +37,27 @@ describe('packageConvert', () => {
   describe('createPackageVersionCreateRequest', () => {
     it('should return a valid request with installationkey and buildinstance', async () => {
       $$.inProject(true);
+      const definitionFile = {
```

Comment:
@smotwani11 given this addition test setup, is there a measurable effect that can be inspected for correctness?

Reply:
All the setup changes go into a zip file string representation:
Pointing you to the comment right above the expect in the same test:

// the most we can assert about VersionInfo because it is a zip file string representation, which changes with time
      expect(typeof request.VersionInfo).to.equal('string');